packages/peregrine/lib/store/actions/checkout/asyncActions.js

```diff
@@ -3,7 +3,7 @@ import { RestApi, Util } from '@magento/peregrine';
 import { closeDrawer } from '../app';
 import { clearCartId, createCart } from '../cart';
 import { getCountries } from '../directory';
-import { getAccountInformation } from '../../selectors/checkoutReceipt';
+import { getAccountInformation } from '@magento/peregrine/lib/store/selectors/checkoutReceipt';
```

Comment:
We could clean up these later to be a little more specific like:

```js
import { selectors } from '@magento/peregrine'
const { checkoutReceipt } = selectors;
```

Reply:
I don't really want to have selectors anyway; these were added without approval in the first place.

Especially now that app state is being provided by context, if we really need to have derived state, we can append derived properties directly to the context value.